src/main/java/seedu/hirehub/model/util/SampleDataUtil.java

```diff
@@ -6,23 +6,32 @@
 
 import seedu.hirehub.model.AddressBook;
 import seedu.hirehub.model.ReadOnlyAddressBook;
+import seedu.hirehub.model.application.Application;
+import seedu.hirehub.model.application.UniqueApplicationList;
 import seedu.hirehub.model.job.Job;
 import seedu.hirehub.model.job.UniqueJobList;
 import seedu.hirehub.model.person.Country;
 import seedu.hirehub.model.person.Email;
 import seedu.hirehub.model.person.Name;
 import seedu.hirehub.model.person.Person;
 import seedu.hirehub.model.person.Phone;
+import seedu.hirehub.model.status.Status;
 import seedu.hirehub.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
+    private static final Person PERSON_1 = new Person(new Name("Alex Yeoh"), new Phone("87438807"),
+        new Email("alexyeoh@example.com"), new Country("SG"), getTagSet("friends"));
+    private static final Job JOB_1 = new Job("Senior Software Engineer (Singapore)",
+        "10 years of experience in C++, Singaporean citizen", 1);
+    private static final Job JOB_2 = new Job("Software Engineer (London)",
+        "Graduated with FCH, UK citizen", 5);
+
```

Comment:
Would it be better to make every job and person a static constant, instead of only declaring two jobs and one person as static constants? Currently I feel its quite inconsistent.

Reply:
only these need to be vars to be referenced in an application though, that would be useless